app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -77,4 +77,15 @@
   blockquote {
     @extend %thredded--blockquote;
   }
+
+  .spoiler {
```

Comment:
Good point Gleb. I'll make that change.

On Mon, May 9, 2016 at 12:17 AM, Gleb Mazovetskiy <notifications@github.com>
wrote:

> In app/assets/stylesheets/thredded/components/_post.scss
> <https://github.com/thredded/thredded/pull/286#discussion_r62449068>:
>
> > @@ -77,4 +77,15 @@
> >    blockquote {
> >      @extend %thredded--blockquote;
> >    }
> > +
> > +  .spoiler {
>
> This should be namespaced, e.g. .thredded--post--content--spoiler.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/286/files/acda6835ac27cb64e55e26fa58fb9bff2bdff8e4#r62449068>
>